src/runtime-rs/crates/resource/src/rootfs/share_fs_rootfs.rs

```diff
@@ -23,14 +23,21 @@ impl ShareFsRootfs {
         share_fs_mount: &Arc<dyn ShareFsMount>,
         cid: &str,
         bundle_path: &str,
-        rootfs: &Mount,
+        rootfs: Option<&Mount>,
+        is_mounted: bool,
     ) -> Result<Self> {
-        let bundle_rootfs = format!("{}/{}", bundle_path, ROOTFS);
-        rootfs.mount(&bundle_rootfs).context(format!(
-            "mount rootfs from {:?} to {}",
-            &rootfs, &bundle_rootfs
-        ))?;
-
+        let bundle_rootfs = if is_mounted {
+            bundle_path.to_string()
```

Comment:
I'm confused about why use bundle_path as the rootfs source.

Reply:
If we use a bundle to start the container, we only need to share the bundle to the shared path between host and guest.

Reply:
I am also confused here. What if the rootfs is not mounted under `bundle/rootfs`, will report an error?
And can mounted nydus-rootfs be supported?

Reply:
The user should give the absolute path of the `rootfs`. I think It should work with nydus.